src/Shared/Recruit.Shared.Web/Configuration/ZendeskConfiguration.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Esfa.Recruit.Shared.Web.Configuration
+{
+    public sealed class ZendeskConfiguration
+    {
+        public string ZenDeskSnippetKey { get; set; }
```

Comment:
Can these properties drop the ZenDesk prefix? They are in the ZendeskConfiguration class so it isn't needed.